packages/react-components/react-tag-picker-preview/src/components/TagPicker/useTagPicker.ts

```diff
@@ -47,9 +47,16 @@ export const useTagPicker_unstable = (props: TagPickerProps): TagPickerState =>
     size: 'medium',
   });
   const onOptionClickBase = state.onOptionClick;
-  state.onOptionClick = useEventCallback(e => {
-    onOptionClickBase(e);
-    state.setOpen(e, false);
+  state.onOptionClick = useEventCallback(event => {
+    onOptionClickBase(event);
+    state.setOpen(event, false);
+  });
+  const setOpenBase = state.setOpen;
+  state.setOpen = useEventCallback((event, newValue) => {
+    if (disabled) {
```

Comment:
is it possible to click on an option when the control is disable? I would have thought the listbox wouldn't even be opened

Reply:
https://github.com/microsoft/fluentui/blob/c61d9935d77e6c1aa72fa4b80ef64683ba1cfff9/packages/react-components/react-combobox/src/components/Combobox/useCombobox.tsx#L72-L82

Seems like this logic is not handled by `useComboboxBaseState`. I can see to include it there to avoid repetition between both `TagPicker` and `Combobox`, but yeah. it seems like this verification is required to ensure the listbox will not be opened